style/properties/longhands/box.mako.rs

```diff
@@ -435,6 +435,7 @@ ${helpers.single_keyword(
     spec="https://drafts.csswg.org/css-transforms/#backface-visibility-property",
     extra_prefixes=transform_extra_prefixes,
     animation_type="discrete",
+    servo_restyle_damage="repaint",
```

Comment:
`backface-visibility` can establish a containing block for all descendants: https://drafts.csswg.org/css-transforms-2/#backface-visibility-property